pkg/controller/trainingjob_controller.go

```diff
@@ -0,0 +1,258 @@
+package controller
+
+import (
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/golang/glog"
```

Comment:
We prefer to use [log15](https://github.com/inconshreveable/log15) recently.

Reply:
Is there any difference between those two log packages?